### ISSUE TITLE
CJK characters turn into unreadable characters while downloading comments

### ISSUE THREAD

**Issue by fishfree** [rwthmoodle/moodle-mod_pdfannotator#17]
After clicking the "Download comments", the downloaded pdf file wronly shows Chinese, Korean and Japanese characters. For your tesing, here are some Chinese characters: 测试评注

**Comment by Friederike-cil** [comment]
We didn't find a font supporting all languages. So we decided to use NotoSans because in our use case we have to support Greek characters.

You can change the font in pdfannotator/shared/index.js in the function openCommentsCallback()

var NotoSans = "Base64-encoded string"; // Add here your custom font.ttf file base64 encoded
doc.addFileToVFS("NotoSans.ttf", NotoSans);
doc.addFont('NotoSans.ttf', 'NotoSans', 'normal');
doc.setFont('NotoSans');

**Comment by fishfree** [comment]
Here is a free font from Google: Droidsansfallback.ttf [DroidSansFallback.zip](https://github.com/rwthmoodle/moodle-mod_pdfannotator/files/3633815/DroidSansFallback.zip).  It's the default font of CJK-region Android release, also having covered western characters. It's a bit big.
After I convert the file to base64-encoded, and replace the all strings "NotoSans" with  "DroidSansFallback" in the index.js file. I cannot add any annotations in a new-created PDF Annotations acvtivity. Error as below:
![pdfannotatinserror](https://user-images.githubusercontent.com/1741341/65292607-a7481680-db8a-11e9-8c75-7c6728ec7c63.png)

**Comment by Friederike-cil** [comment]
I have just tested it and could not reproduce this error. Does it still occur?

**Comment by synnac** [comment]
yes, when comments are exported, the pdf file is in messy codes.

**Comment by nisaDoc** [comment]
Hello @fishfree, @synnac,
thank you guys for reporting this to us.
Could you tell us the steps you have done? It would be help full for us to reproduce the errors.

Best regards,
Nisa
